### PR TITLE
EPAS minor upgrades: replaced `yum` with `dnf`

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

`yum` package manager was replaced by `dnf`. It seems like we updated all installation topics, but we did not update the minor upgrade doc to reflect this. 

https://enterprisedb.atlassian.net/browse/DOCS-802

### CODE REVIEW

**Review by tusharahuja — APPROVED** [review]
looks good. Thanks.

**Review by theadamwright — APPROVED** [review]

File: product_docs/docs/epas/16/upgrading/05_performing_a_minor_version_update_of_an_rpm_installation.mdx [hunk 1]
```diff
@@ -5,28 +5,32 @@ redirects:
...
-```
+    Where `package_name` is the name of the package you want to update. Include wildcard values in the update command to update multiple related packages with a single command. For example, use the following command to update all packages with names that include the expression `edb`:
+
+    ```shell
+    dnf update edb*
+    ```
 
-!!! Note
-    The `yum update` command performs an update only between minor releases. To update between major releases, use `pg_upgrade`.
+    !!! Note
```
Comment (djw-m) [inline product_docs/docs/epas/16/upgrading/05_performing_a_minor_version_update_of_an_rpm_installation.mdx:30]: I'd move this warning to the top. This process is only useful for minor upgrades, major upgrades need pg_upgrade.

File: product_docs/docs/epas/16/upgrading/05_performing_a_minor_version_update_of_an_rpm_installation.mdx [hunk 2]
```diff
@@ -5,28 +5,32 @@ redirects:
...
 
-Where `package_name` is the name of the package you want to update. Include wildcard values in the update command to update multiple related packages with a single command. For example, use the following command to update all packages whose names include the expression `edb`:
+    ```shell
+    dnf update <package_name>
+    ```
 
-```shell
-yum update edb*
-```
+    Where `package_name` is the name of the package you want to update. Include wildcard values in the update command to update multiple related packages with a single command. For example, use the following command to update all packages with names that include the expression `edb`:
```
Comment (josh-heyer) [inline product_docs/docs/epas/16/upgrading/05_performing_a_minor_version_update_of_an_rpm_installation.mdx:24]: nit-pick and existed prior to this edit, but... This is slightly misleading. Should be,

```suggestion
    Where `package_name` is the name of the package you want to update. Include wildcard values in the update command to update multiple related packages with a single command. For example, use the following command to update all packages with names start with `edb`:
```

For symmetry, might also include same example for step 1 above (ideally you'd know which packages an expression will update before updating them).

**Review by djw-m — COMMENTED** [review]
LGTM

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**